### ISSUE TITLE
404 page directive not working

### ISSUE THREAD

**Issue by johnb8005** [lipanski/docker-static-website#9]
I tried to implement the E404 directive but I can't make it work.

My starting point was: https://github.com/lipanski/docker-static-website#how-can-i-overwrite-the-default-error-pages

Repo where I tried to implement it:
https://github.com/nexys-system/boilerplate_vite

* `Dockerfile`: https://github.com/nexys-system/boilerplate_vite/blob/master/Dockerfile
* `httpd.conf`: https://github.com/nexys-system/boilerplate_vite/blob/master/httpd.conf

 but when I start the container I get the following warnings:

```
2022-08-08T15:12:14.754050482Z httpd: config error 'E404:/home/static/index.html' in 'httpd.conf'
2022-08-08T15:12:14.754090667Z httpd: config error 'E404:index.html' in 'httpd.conf'
```

Note that when I use basic auth or the `I` directive, the `httpd.conf` is interpreted and I see the expected results.

I also looked into the [busybox source code](https://git.busybox.net/busybox/tree/networking/httpd.c#n853) but, unless `ENABLE_FEATURE_HTTPD_ERROR_PAGES ` is disabled (which I don't think it is [see here](https://github.com/lipanski/docker-static-website/blob/master/.config#L870)), it should be ok

Any ideas?

**Comment by lipanski** [comment]
Fixed in [this commit](https://github.com/lipanski/docker-static-website/commit/3f40928dc1124167696278b01da9e17e0b3de3f6) and released as `latest` and `2.1.0`.

The issue was the following: "Error pages are only parsed on the main config file". What this means is that the conf file has to be passed to `httpd` via the `-c` argument. That's how you point to the main file.

In order to avoid having users of the Dockerfile to include this every time, I added a blank `httpd.conf` as the main file. You can simply include an override in your project root and that's it:

```
FROM lipanski/docker-static-website:latest

# Copy your static files
COPY . .
```

Or alternatively you can be explicit about the `CMD` line:

```
FROM lipanski/docker-static-website:latest

# Copy your static files
COPY . .

CMD ["/busybox", "httpd", "-f", "-v", "-p", "3000", "-c", "httpd.conf"]
```

Also note that the error page path should be relative, like `E404:e404.html` or `E404:some-subdirectory/e404.html`.

**Comment by johnb8005** [comment]
Thank you. I could successfully update my project too
